packages/react/src/components/SuggestionMenu/DefaultRenderItems.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  SuggestionMenuItem,
+  SuggestionMenuItemProps,
+} from "./SuggestionMenuItem";
+import {
+  SuggestionMenuLabel,
+  SuggestionMenuLabelProps,
+} from "./SuggestionMenuLabel";
+import { SuggestionMenu } from "./SuggestionMenu";
+
+export type SuggestionMenuItemsProps = {
+  items: (SuggestionMenuItemProps | SuggestionMenuLabelProps)[];
+  loadingState?: "initial" | "loading-initial" | "loading" | "loaded";
+  selectedIndex?: number;
+};
+
+export function DefaultRenderItems(props: SuggestionMenuItemsProps) {
+  let numLabels = 0;
+
+  return (
+    <SuggestionMenu loadingState={props.loadingState}>
+      {props.items.map((item, index) => {
```

Comment:
we need to add a `key` to children, pretty sure you'll get a react warning otherwise (see console)